followthemoney/types/common.py

```diff
@@ -183,6 +184,7 @@ def to_dict(self) -> PropertyTypeToDict:
         data: PropertyTypeToDict = {
             "label": gettext(self.label),
             "plural": gettext(self.plural),
+            "description": gettext(self.docs),
```

Comment:
We currently don’t translate the descriptions and I’m not sure if we ever will, but it doesn’t hurt either…